SOFTWARE/AfterlifeLightKit/Settings.h

```diff
@@ -0,0 +1,65 @@
+#define POWERCELL_PIN 0
+#define CYCLOTRON_PIN 5
+
+enum Directions
+{
+    FORWARD,
+    REVERSE
+};
+
+enum MODES
+{
+    CLASSIC,
+    AFTERLIFE,
+    TVG,
+    REBOOT,
+};
+
+struct LightStrip
+{
+    int length;
+    // GRB/RGB
+    // WS2812B/WS2811 (or FastLed type?)
+};
+
+struct HardwareConfiguration
+{
+    // Default length of a Powercell should be 15 LEDs
+    LightStrip Powercell = {15};
+    // Default length of Cyclotron should be 60 LEDs
+    LightStrip Cyclotron = {60};
+};
+
+struct RGBColor {
```

Comment:
Hoping we can eventually remove this in favour of CRGB from FastLED, but I couldn't get it working:
```ino
struct PowercellSettings
{
    Directions direction;
    int speed;
    CRGB color;
};
```
(I can't remember the exact compiler error I got)